circle.yml

```diff
@@ -1,3 +1,11 @@
 machine:
   java:
-    version: oraclejdk8
\ No newline at end of file
+    version: oraclejdk8
+
+dependencies:
+  override:
```

Comment:
This should probably be `pre`, otherwise it overrides Circle's inferred dependency commands. It didn't fail for you because there was already cache there.